Orange/distance/__init__.py

```diff
@@ -194,26 +202,32 @@ def fit(self, data, axis=1):
             x = x.T
         n, m = x.shape
         if n <= m:
-            raise ValueError('Too few observations for the number of dimensions.')
+            raise ValueError('Too few observations for the number of '
+                             'dimensions.')
         self.axis = axis
         self.VI = np.linalg.inv(np.cov(x.T))
 
     def __call__(self, e1, e2=None, axis=None, impute=False):
-        assert self.VI is not None, "Mahalanobis distance must be initialized with the fit() method."
+        assert self.VI is not None, \
+            "Mahalanobis distance must be initialized with the fit() method."
 
         x1 = _orange_to_numpy(e1)
         x2 = _orange_to_numpy(e2)
 
         if axis is not None:
-            assert axis == self.axis, "Axis must match its value at initialization."
+            assert axis == self.axis, \
+                "Axis must match its value at initialization."
         if self.axis == 0:
             x1 = x1.T
             if x2 is not None:
                 x2 = x2.T
-        if x1.shape[1] != self.VI.shape[0] or x2 is not None and x2.shape[1] != self.VI.shape[0]:
+        if not x1.shape[1] == self.VI.shape[0] or \
+                x2 is not None and not x2.shape[1] == self.VI.shape[0]:
             raise ValueError('Incorrect number of features.')
 
-        dist = skl_metrics.pairwise.pairwise_distances(x1, x2, metric='mahalanobis', VI=self.VI)
+        dist = skl_metrics.pairwise.pairwise_distances(x1, x2,
+                                                       metric='mahalanobis',
+                                                       VI=self.VI)
```

Comment:
This is another, better case agains aligning with parentheses. 

```
        dist = skl_metrics.pairwise.pairwise_distances(
            x1, x2, metric='mahalanobis', VI=self.VI)
```

is shorted, more compact and better.